static/app/components/updatedEmptyState.tsx

```diff
@@ -143,32 +41,83 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     }
   );
 
+  const {isLoading: isLoadingRegistry, data: registryData} =
+    useSourcePackageRegistries(organization);
+
+  const currentPlatformKey = project?.platform ?? 'other';
+  const currentPlatform = platforms.find(
+    p => p.id === currentPlatformKey
+  ) as PlatformIntegration;
+
+  const platformPath = getPlatformPath(currentPlatform);
+
+  useEffect(() => {
+    async function getGettingStartedDoc() {
+      const mod = await import(
+        /* webpackExclude: /.spec/ */
+        `sentry/gettingStartedDocs/${platformPath}`
+      );
+      setModule(mod);
+    }
+    getGettingStartedDoc();
+    return () => {
+      setModule(null);
+    };
+  }, [platformPath]);
+
   if (
-    !platformGuidedSteps ||
     !project ||
     projectKeysIsError ||
     projectKeysIsLoading ||
     !projectKeys ||
-    projectKeys.length === 0
+    projectKeys.length === 0 ||
+    !module ||
+    !currentPlatform
   ) {
     return null;
   }
 
   const dsn = projectKeys[0].dsn.public;
-  const language = project?.platform === 'node' ? 'javascript' : 'python';
-
+  const {default: docs} = module;
+
+  const docParams: DocsParams<any> = {
```

Comment:
Does this have to be `any`?

Reply:
it's not great but that's what we have across a bunch of components right now, i can come back and see if it really is necessary after this pr